File: C/numeros_pares.c
#include <stdio.h>

int main()
{
    int n, qtdPares;

    printf("Quantos numeros voce vai digitar? ");
    scanf("%d", &n);

    int numeros[n];

    for(int i = 0; i < n; i++)
    {
        printf("Digite um numero: ");
        scanf("%d", &numeros[i]);
    }
    printf("\nNUMEROS PARES:\n");
    qtdPares = 0;
    for(int i = 0; i < n; i++)
    {
        if (numeros[i] % 2 == 0)
        {
            printf("%d ", numeros[i]);
            qtdPares++;
        }
    }
    printf("\n\nQUANTIDADE DE PARES = %d", qtdPares);

    return 0;
}

File: Python/numeros_pares.py
qtdPares: int
n = int(input("Quantos numeros voce vai digitar? "))

vet = [0 for x in range(n)]

for i in range(0, n):
    vet[i] = int(input("Digite um numero: "))

print("\nNUMEROS PARES:")
qtdPares = 0
for i in range(0, n):
    if vet[i] % 2 == 0:
        print(f"{vet[i]} ", end="")
        qtdPares = qtdPares + 1

print(f"\n\nQUANTIDADE DE PARES = {qtdPares}")


